src/Microsoft.SqlTools.CoreServices/Connection/ReliableConnection/SqlConnectionHelperScripts.cs

```diff
@@ -46,6 +46,6 @@ @rc 			int
 SELECT @filepath AS FilePath
 ";
 
-    public const string GetOsVersion = @"SELECT OSVersion = RIGHT(@@version, LEN(@@version)- 3 -charindex (' ON ', @@version))";
+    public const string GetOsVersion = @"SELECT OSVersion = RIGHT(@@version, LEN(@@version)- 3 -charindex (' on ', @@version))";
```

Comment:
is there a way to write this so it's case-insensitive?  Perhaps adding a `tolower` call somewhere so we can be sure this condition won't start failing in the future?

Reply:
done